numpy/core/src/umath/override.c

```diff
@@ -180,18 +179,6 @@ normalize_reduceat_args(PyUFuncObject *ufunc, PyObject *args,
 }
 
 static int
-normalize_outer_args(PyUFuncObject *ufunc, PyObject *args,
-                    PyObject **normal_args, PyObject **normal_kwds)
-{
```

Comment:
But I already combine `reduce` and `accumulate` similarly. But let me at least adjust the function name accordingly.